tests/bit_vector/get_test.cu

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <utils.hpp>
+
+#include <cuco/detail/trie/bit_vector/bit_vector.cuh>
+
+#include <thrust/device_vector.h>
+#include <thrust/execution_policy.h>
+
+#include <catch2/catch_test_macros.hpp>
+
+template <class BitVectorRef>
+__global__ void get_kernel(BitVectorRef ref, size_t n, uint64_t* output)
+{
+  size_t index  = blockIdx.x * blockDim.x + threadIdx.x;
+  size_t stride = gridDim.x * blockDim.x;
+  while (index < n) {
+    output[index] = ref.get(index);
+    index += stride;
+  }
+}
+
+bool modulo_bitgen(uint64_t i) { return i % 7 == 0; }
```

Comment:
Ah, I see. So after linking this function should be available to the other test.